### ISSUE TITLE
how implement memory efficient DenseNet  using Tensorflow? 

### ISSUE THREAD

**Issue by phybrain** [gpleiss/efficient_densenet_pytorch#25]
is there a master can  convet pytorch implementation to tf？

**Comment by taineleau** [comment]
Please read our technical report first. https://arxiv.org/pdf/1707.06990.pdf
Basically, the efficient DenseNet requires manually allocating memory for the output of BN/conv/concat.

**Comment by phybrain** [comment]
I have read the report ，however I can't find a way to implement  allocating-memory  concat  in  tensorflow

**Comment by taineleau** [comment]
@phybrain alas, so currently I don't think you can do it in Tensorflow.

**Comment by joeyearsley** [comment]
I've recreated the tensorflow version using gradient checkpointing like this repo does now.
https://github.com/joeyearsley/efficient_densenet_tensorflow

**Comment by gpleiss** [comment]
Awesome! Thanks @joeyearsley ! I'll link to it in the readme.
